src/components/PostDetails/PostDetails.jsx

```diff
@@ -1,45 +1,124 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
 import { NewCommentForm } from '../NewCommentForm';
+import { Loader } from '../Loader';
+
 import './PostDetails.scss';
 
-export const PostDetails = () => (
-  <div className="PostDetails">
-    <h2>Post details:</h2>
-
-    <section className="PostDetails__post">
-      <p>sunt aut facere repellat provident occaecati excepturi optio</p>
-    </section>
-
-    <section className="PostDetails__comments">
-      <button type="button" className="button">Hide 2 comments</button>
-
-      <ul className="PostDetails__list">
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>My first comment</p>
-        </li>
-
-        <li className="PostDetails__list-item">
-          <button
-            type="button"
-            className="PostDetails__remove-button button"
-          >
-            X
-          </button>
-          <p>sad sds dfsadf asdf asdf</p>
-        </li>
-      </ul>
-    </section>
-
-    <section>
-      <div className="PostDetails__form-wrapper">
-        <NewCommentForm />
-      </div>
-    </section>
-  </div>
-);
+import {
+  deleteComment,
+  getPostComments,
+  addComment,
+} from '../../api/comments';
+
+import { getPostDetails } from '../../api/posts';
+
+export const PostDetails = ({ selectedPostId }) => {
+  const [selectedPost, setSelectedPost] = useState(null);
+  const [comments, setComments] = useState(null);
+  const [commentsHidden, setCommentsHidden] = useState(false);
+
+  useEffect(() => {
+    getPostDetails(selectedPostId)
+      .then(result => setSelectedPost(result));
+
+    setComments(null);
+
+    getPostComments(selectedPostId)
```

Comment:
You could create another function for this operation and call it smth like `updateComments`